target/linux/lantiq/xrx200/base-files/etc/board.d/02_network

```diff
@@ -47,6 +47,9 @@ lantiq_setup_interfaces()
 	avm,fritz7430)
 		ucidef_add_switch "switch0" \
 			"2:lan:3" "3:lan:4" "4:lan:1" "5:lan:2" "6t@eth0"
+	avm,fritz3390)
+		ucidef_add_switch "switch0" \
+			"0:lan:3" "1:lan:4" "2:lan:2" "4:lan:1" "5:lan:5" "6t@eth0"
```

Comment:
Is port number 5 connected to WiFi?

Reply:
Yes, from what I understood it seems to be the connection to the second SOC (also know as WASP-interface)

Reply:
I would remove LAN5 entirely for now, as it causes more confusion than it serves - see my other comments.